conf/st

```diff
@@ -68,7 +69,7 @@ stop() {
 
 _init() {
     lnet_up
-    if [ "$XPRT" = "lnet" ]; then
+    if [[ "$XPRT" == "lnet" || "$lnet_available" == "true" ]]; then
```

Comment:
Why do you need `"$XPRT" == "lnet"` term? Lnet module can be loaded only when it is available anyway.